bundle/manifests/ibm-user-management-operator.clusterserviceversion.yaml

```diff
@@ -124,7 +125,7 @@ spec:
                   value: icr.io/cpopen/cpfs/api_service:1.0.0
                 - name: RELATED_IMAGE_INSTANCE_MANAGEMENT_SERVICE
                   value: icr.io/cpopen/cpfs/instance_management_service:1.0.0
-                image: icr.io/cpopen/ibm-user-management-operator:latest
+                image: quay.io/bedrockinstallerfid/ibm-user-management-operator:dev
```

Comment:
Hi @qpdpQ 
Can you try to run `make clean-before-commit` to clean those test-specific changes?